IdnoPlugins/IndiePub/Pages/MicroPub/Endpoint.php

```diff
@@ -130,6 +163,48 @@ function post()
 
                 // Get an appropriate plugin, given the content type
                 if ($contentType = ContentType::getRegisteredForIndieWebPostType($type)) {
+                    if ($type == 'photo' && empty($name) && !empty($content)) {
+                        $name    = $content;
+                        $content = '';
+                    }
```

Comment:
i'm a little confused what happened here; this partially duplicates code from line ~102